### PR TITLE
Adds RecentLinks to homepage.

### PR DESCRIPTION
**Pull request by ijpatricio**
Addresses #118

GraphQL query updated.
RecentLinks component added. Prop for amount of recent items to display.
Added the compoenent to the homepage.

@philsturgeon should I add tags as badges?
image

Preview picture:
screencapture-localhost-8000-2020-01-27-21_40_07

### CODE REVIEW

File: src/components/RecentLinks.jsx [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+
+    return (
+        <div>
+            <div>
+                <p className="h2"> Recent Links</p>
+            </div>
+            {
+                recentLinks.map(link => (
+                    <Fragment>
+                        <a href={link.url} target="_blank" class="py-3 pl-8 pr-2 border-bottom d-block text-decoration-none">
```
Comment (mbifulco) [inline src/components/RecentLinks.jsx]: Hey there - I think you should be able to remove the `<Fragment>` surrounding this link, and I think you may need to add a `key={something}` prop to [keep react happy](https://reactjs.org/docs/lists-and-keys.html) here

File: src/components/RecentLinks.jsx [hunk 2]
```diff
@@ -0,0 +1,35 @@
+import React, { Fragment } from 'react';
+
+import Col from 'react-bootstrap/Col';
+import Row from 'react-bootstrap/Row';
+
+import { graphql } from 'gatsby';
+
+export default ({ links, showAmmount }) => {
```
Comment (mbifulco) [inline src/components/RecentLinks.jsx]: A small typo here - amount should only have one "m"

File: src/components/RecentLinks.jsx (hunk 1 above)
Comment (ijpatricio) [reply src/components/RecentLinks.jsx]: Hey! Sure. 
- `Fragment` removed.
- So the `key` is to give a unique name, and on native html it is not needed. But I went ahead and abstracted a RecentLinkItem, which is keyed.

Yes, it makes more sense this way now.

Let me know your thoughts!

File: src/components/RecentLinks.jsx (hunk 2 above)
Comment (ijpatricio) [reply src/components/RecentLinks.jsx]: Done :)

**Comment by philsturgeon** [comment]
This is amazing! Merging :) 

Could we use this style for the existing links on categories and tags pages as a follow up PR? Also, yes add the tags as badges to this list and the categories list please.

**Pull request merged**